package.json

```diff
@@ -7,7 +7,7 @@
         "dev": "vite",
         "build": "tsc && vite build",
         "lint": "eslint --ext ts,tsx --report-unused-disable-directives --max-warnings 0 ./src",
-        "lint:fix": "eslint --ext ts,tsx --report-unused-disable-directives --max-warnings 0  --fix",
+        "lint:fix": "eslint --fix --ext ts,tsx --report-unused-disable-directives --max-warnings 0 ./src",
```

Comment:
Egor removed ./src so that the pre-commit linter only checks files that have been changed, not the entire project